js/libs/ui-shared/src/user-profile/OptionsComponent.tsx

```diff
@@ -25,7 +33,7 @@ export const OptionComponent = (props: UserProfileFieldProps) => {
                 key={option}
                 id={option}
                 data-testid={option}
-                label={option}
+                label={props.t(unWrap(optionLabel?.on || option))}
```

Comment:
Do we need to `unWrap(label)` here as well, or can we  get away with:
```suggestion
                label={props.t(optionLabel?.on ? unWrap(optionLabel.on) : option)}
```

Reply:
No, it can be a translation key, see the issue